<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>migration</class>
 <widget class="QMainWindow" name="migration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>migration form</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-227</y>
         <width>780</width>
         <height>852</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="3" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;b&gt;Place of Origin:&lt;/b&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="registrationNoLabel">
            <property name="text">
             <string>Registration No</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="registrationNoLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="districtLabel_2">
            <property name="text">
             <string>District</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="districtLineEdit_2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="municipalityLabel_2">
            <property name="text">
             <string>Municipality</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="municipalityLineEdit_2"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="wardNoLabel_2">
            <property name="text">
             <string>Ward No</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="wardNoLineEdit_2"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="roadStreetLabel_2">
            <property name="text">
             <string>Road/Street</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="roadStreetLineEdit_2"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="houseNoLabel_2">
            <property name="text">
             <string>House No</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="houseNoLineEdit_2"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="villageCommunityLabel">
            <property name="text">
             <string>Village/Community</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="villageCommunityLineEdit"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="nameOfHomeownerWhoRelocatedLabel">
            <property name="text">
             <string>Name of homeowner who relocated</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="nameOfHomeownerWhoRelocatedLineEdit"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="dateOfRelocatedLabel">
            <property name="text">
             <string>Date of relocated</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QDateEdit" name="dateOfRelocatedDateEdit"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="reasonOfRelocatedLabel">
            <property name="text">
             <string>Reason of relocated</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QComboBox" name="reasonOfRelocatedComboBox">
            <item>
             <property name="text">
              <string>job</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>bussiness</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>new house</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>other</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
        <item row="6" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <column>
           <property name="text">
            <string>Birth certificate no</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Full Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Gender</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Citizenship No</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Issue date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Issue District</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Relation with informative</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout_4">
          <item row="0" column="0">
           <widget class="QLabel" name="registrationNoLabel_2">
            <property name="text">
             <string>&lt;b&gt;Registration No&lt;/b&gt;</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="registrationNoLineEdit_2"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="familyRecordFormNoLabel">
            <property name="text">
             <string>&lt;b&gt;Family Record Form No</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="familyRecordFormNoLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="districtLabel">
            <property name="text">
             <string>District</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="districtLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="municipalityLabel">
            <property name="text">
             <string>Municipality</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="municipalityLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="wardNoLabel">
            <property name="text">
             <string>Ward No</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="wardNoLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="roadStreetLabel">
            <property name="text">
             <string>Road/Street</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="roadStreetLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="houseNoLabel">
            <property name="text">
             <string>House No</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="houseNoLineEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="villageLabel">
            <property name="text">
             <string>Village/Community</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="villageyLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;b&gt;Place of Destination:&lt;/b&gt;</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <layout class="QFormLayout" name="formLayout_5">
          <item row="0" column="0">
           <widget class="QLabel" name="InformationProvidedByLabel">
            <property name="text">
             <string>&lt;b&gt;Information Provided By</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="nameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="nameLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="citizenshipCertificateNoLabel">
            <property name="text">
             <string>Citizenship Certificate No.</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="citizenshipCertificateNoLineEdit"/>
          </item>
         </layout>
        </item>
        <item row="5" column="0">
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="bDetailsOfThePersonRelocatedLabel">
            <property name="text">
             <string>&lt;b&gt;Details Of The Person Relocated</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="addMore">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
